modules/mono/glue/GodotSharp/GodotSharp/Core/Attributes/AssemblyHasScriptsAttribute.cs

```diff
@@ -8,25 +10,28 @@ namespace Godot
     [AttributeUsage(AttributeTargets.Assembly)]
     public class AssemblyHasScriptsAttribute : Attribute
     {
-        private readonly bool requiresLookup;
-        private readonly System.Type[] scriptTypes;
+        public bool RequiresLookup { get; private set; }
+        public Type[]? ScriptTypes { get; private set; }
```

Comment:
Same reason as why `readonly` is no longer allowed. Previously, we used reflection to assign these properties, and that gave us access to the backing field. We can't do that from code, as the backing field is hidden at that level.

Reply:
But in this case this class is an attribute, if we are only using the attributes from the source generators that shouldn't matter, right? In fact, we technically wouldn't need the properties at all since the source generator would only be parsing the ctor declaration.

Reply:
Ah, you're right. This must have been a mistake.